coralogix-eventbridge-policy/README.md

```diff
@@ -0,0 +1,10 @@
+# EventBridge Policy
+
+The module will create a policy on a given EventBridge Event Bus so that Coralogix can send data to it.
+
+## Fields
+
+| Parameter          | Description                                                                                   | Default Value | Required |
+|--------------------|-----------------------------------------------------------------------------------------------|---------------|----------|
```

Comment:
You are missing param to give the customer a way to select the predefined coralogix regions